code/_core/obj/item/weapon/ranged/unique/_unique.dm

```diff
@@ -0,0 +1,20 @@
+/obj/item/weapon/ranged/unique/
```

Comment:
Not sure I like this tbh. I rather you make a subtype for a weapon that takes reagents as ammo instead of a catch-all type that takes anything.

Reply:
Probably just me overthinking weapon ideas and preparing for them, I'm good with unique weapons, but for now, poof. It shall disappear.